istantestd.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+NPM=$(which npm)
+ROOTPATH=$(pwd)
+$NPM install
+echo "Cloning test.."
+git clone -b authentication https://github.com/OADA/oada-compliance.git 
+cd oada-compliance/
+$NPM install
+
+cd $ROOTPATH
+echo "Starting instrumented server.."
+export PORT=8443
+istanbul cover index.js & 
```

Comment:
(My) style preference:

PORT=8443 istanbul cover index.js &

Unless somehow PORT communicates the port to `./test`. If so, I think I would rather: `./test https://localhost:8443`